specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/examples/AFDEndpoints_Update.json

```diff
@@ -19,12 +19,12 @@
         "id": "/subscriptions/subid/resourcegroups/RG/providers/Microsoft.Cdn/profiles/profile1/afdendpoints/endpoint1",
         "type": "Microsoft.Cdn/profiles/afdendpoints",
         "tags": {},
-        "location": "WestCentralUs",
+        "location": "global",
         "properties": {
-          "hostName": "endpoint1.azureedge.net",
+          "hostName": "endpoint1-ezh7ddcmguaeajfu.azureedge.net",
```

Comment:
should we update it to "z01.azurefd"

Reply:
updated. I also added "autoGeneratedDomainNameLabelScope" which was missed in this particular file.